### PR TITLE
allow to select upgrades with step != 1

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: roles/forklift_versions/library/forklift_versions.py [hunk 1]
```diff
@@ -24,8 +24,12 @@
     description: scenario version to look up
     required: True
   scenario_os:
-    descrption: scenario OS to look up
+    description: scenario OS to look up
     required: True
+  scenario_step:
```
Comment (evgeni) [inline roles/forklift_versions/library/forklift_versions.py:29]: bikeshed: `upgrade_step`?

File: roles/forklift_versions/library/forklift_versions.py [hunk 2]
```diff
@@ -24,8 +24,12 @@
     description: scenario version to look up
     required: True
   scenario_os:
-    descrption: scenario OS to look up
+    description: scenario OS to look up
     required: True
+  step:
```
Comment (evgeni) [inline roles/forklift_versions/library/forklift_versions.py:29]: bikeshed: `upgrade_step`?

File: roles/forklift_versions/molecule/default/verify.yml [hunk 3]
```diff
@@ -38,6 +38,22 @@
...
 
+    - name: "Include forklift_versions for upgrades with step=2"
+      include_role:
+        name: "forklift_versions"
+      vars:
+        scenario: "{{ pipeline_type }}"
+        scenario_os: "{{ pipeline_os }}"
+        scenario_version: "3.16"
+        upgrade: True
+        scenario_step: 2
```
Comment (ekohl) [inline roles/forklift_versions/molecule/default/verify.yml:49]: This doesn't match the `step` used everywhere.

File: roles/forklift_versions/library/forklift_versions.py (hunk 2 above)
Comment (ekohl) [reply roles/forklift_versions/library/forklift_versions.py:29]: Yes, I think that would make more sense.

File: roles/forklift_versions/molecule/default/verify.yml (hunk 3 above)
Comment (evgeni) [reply roles/forklift_versions/molecule/default/verify.yml:49]: the playbook in the role uses `scenario_step` that gets passed to the `step` param, but yeah, alligning those is nice

**Pull request merged**